styles/components/alert/alert.css

```diff
@@ -72,10 +72,10 @@
   color: var(--primary-900);
   background-color: var(--primary-200);
 }
-
+/* 
 .alert-primary .alert-close{
   filter: invert(35%) sepia(27%) saturate(4472%) hue-rotate(177deg) brightness(96%) contrast(98%);
-}
+} */
```

Comment:
You could try avoid adding commented code in the PR. It's a bad practice as suggested by our MA Pranshu bhaiya.